ggshield/cmd/iac/scan/iac_scan_utils.py

```diff
@@ -36,7 +36,9 @@ def create_output_handler(ctx: click.Context) -> IaCOutputHandler:
     else:
         output_handler_cls = IaCTextOutputHandler
     config: Config = ctx.obj["config"]
-    return output_handler_cls(verbose=config.user_config.verbose)
+    return output_handler_cls(
+        verbose=config.user_config.verbose, debug=config.user_config.debug
```

Comment:
Is it something we want to add debug here ?

Reply:
I'm not sure what this is for, I think it comes from Paul's MR which I rebased on.

Reply:
@Paul-GitGuardian ?

Reply:
This was added in #800 
we pass the debug option to the output handler to optionally display some warnings based on it